components/support/rc_crypto/nss/src/pk11/sym_key.rs

```diff
@@ -91,3 +95,23 @@ pub(crate) fn import_sym_key(
         ))
     }
 }
+
+/// Safe wrapper around PK11_PBEKeyGen that
+/// de-allocates memory when the key goes out of
+/// scope.
+pub(crate) fn pbe_key_gen(
```

Comment:
I think that's fine to inline into the caller, since there's only one use of this.